tests/test_trello_service.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from unittest import mock
+
+from application.dashboard.trello_service import trello_service
+
+
+# Mock out the Trello client at class level to be sure we don't make any calls out to external Trello API
+@mock.patch.object(trello_service, "client")
+class TestTrelloService:
+    def test_client_is_mocked(self, client):
+        trello_service.get_measure_cards()
+        client.get_board.assert_called_with(mock.ANY)
+
+    @pytest.mark.parametrize(
+        "card_name, expected_output",
+        (
+            ("Just a title", "Just a title"),
+            ("[ABC 123] Here's a title", "Here's a title"),
+            ("[AB[C[[ 123] Here's a title", "Here's a title"),
+            ("[ABC 123] [Here's] a title", "[Here's] a title"),
+            ("[ABC []123] Here's a title", "123] Here's a title"),
```

Comment:
@TheDoubleK heh, I think I’d sort-of expect this to strip out the entirety of `[ABC []123]` – but it's such an unlikely scenario I'm not sure we need to write a parser for this... ;-)

Reply:
Hmm, I think all we'd need to do would be change `+?` to `+*` in the regex.  So we could go that way?

Although that would change the output for the previous test case, ie `"[ABC 123] [Here's] a title"` would give you `"a title"` as the result.

I'm not doing bracket pair matching.